### PR TITLE
Naming db browser for sq lite

### PR DESCRIPTION
**Pull request by horst-p-w-neubauer**
Suggestion for consistent naming.
Before merging, the .ts files should be updated.

### CODE REVIEW

**Comment by horst-p-w-neubauer** [comment]
confused - my commit shows only 2 changed files :-(

**Comment by horst-p-w-neubauer** [comment]
I close it.

**Comment by justinclift** [comment]
> confused - my commit shows only 2 changed files :-(

It's probably just a mix up with the branch.  If you swap to the master branch (eg: `git checkout master`), then create a new branch just for the naming change from that (eg: `git checkout -b naming_change_v1`) it should give you a clean slate to work from. :smile:

File: src/MainWindow.ui [hunk 1]
```diff
@@ -1643,7 +1643,7 @@ You can drag SQL statements from the Schema column and drop them into the SQL ed
     <string>Save the current session to a file</string>
    </property>
    <property name="whatsThis">
-    <string>This button lets you save all the settings associated to the open DB to a DB4S project file</string>
+    <string>This button lets you save all the settings associated to the open DB to a DB Browser for SQLite project file</string>
```
Comment (justinclift) [inline src/MainWindow.ui:1646]: This looks like an easy enough change. :smile:

File: src/MainWindow.ui [hunk 2]
```diff
@@ -1664,7 +1664,7 @@ You can drag SQL statements from the Schema column and drop them into the SQL ed
     <string>Load a working session from a file</string>
    </property>
    <property name="whatsThis">
-    <string>This button lets you open a DB4S project file</string>
+    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This button lets you open a DB Browser for SQLite project file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (justinclift) [inline src/MainWindow.ui:1667]: The `DB4S` -> `DB Browser for SQLite` bit here is easy too.  Not sure about the additional html tags though, are they on purpose?  It's fine if they are, I'm just not sure, so asking. :smile:

File: src/MainWindow.ui [hunk 3]
```diff
@@ -174,8 +174,8 @@ You can drag SQL statements from an object row and drop them into other applicat
             <rect>
              <x>0</x>
              <y>0</y>
-             <width>596</width>
-             <height>526</height>
+             <width>692</width>
+             <height>489</height>
```
Comment (justinclift) [inline src/MainWindow.ui:178]: Are the height+width changes here on purpose? :smile:

File: src/MainWindow.ui [hunk 4]
```diff
@@ -744,7 +744,7 @@ You can drag SQL statements from an object row and drop them into other applicat
      <x>0</x>
      <y>0</y>
      <width>1037</width>
-     <height>22</height>
+     <height>21</height>
```
Comment (justinclift) [inline src/MainWindow.ui:747]: Same question here.  Is this height change on purpose, or accidental? :smile:

**Pull request closed without merging**